### PR TITLE
Update change log

### PR DESCRIPTION
**Pull request by igoralmeida**
Summarizing: https://github.com/syl20bnr/spacemacs/commits/develop?before=69a89cc41243cfb189238a2297fde4498d7fcf6d+928
Starting with: 79df0576545c4398a0ed7c14099cd877faf64095

New pointer: https://github.com/syl20bnr/spacemacs/commits/develop?before=69a89cc41243cfb189238a2297fde4498d7fcf6d+893
Start with: 25fa9b0a50f04d8375529ea492b0413cb2f435cf

### CODE REVIEW

File: CHANGELOG.develop [hunk 1]
```diff
@@ -230,6 +233,8 @@ a sane way, here is the complet list of changed key bindings:
   Benner)
 - Handle backward argument better when cycling themes (thanks to bmag)
 - Remove wizard question about completion engine. (thanks to Sylvain Benner)
+- Conventions: add SPC m f for web frameworks (thanks to Sylvain Benner)
```
Comment (randomwangran) [inline CHANGELOG.develop]: using kbd mark?

~SPC m f~

File: CHANGELOG.develop [hunk 2]
```diff
@@ -346,6 +351,25 @@ a sane way, here is the complet list of changed key bindings:
   - Fix switch to home buffer moves prev buffers cursor (thanks to duianto)
   - Correct undo-tree buffer name in =spacemacs-visual= (thanks to Winkey Wong)
   - Update renamed helm split window variable (thanks to duianto)
+  - Fix using eyebrowse without persp-mode (thanks to Steven Allen)
```
Comment (randomwangran) [inline CHANGELOG.develop]: =persp-mode=

? consistency?

File: CHANGELOG.develop [hunk 3]
```diff
@@ -406,6 +436,8 @@ a sane way, here is the complet list of changed key bindings:
 - Add =cmake-ide= (thanks to Alexander Dalshov)
 - Fixed a typo by moving one closing parenthesis (thanks to Richard Kim)
 - Add option to control cmake-ide package (thanks to Alexander Dalshov)
+- Fix =google-set-c-style= functions being added regardless of config (thanks to
```
Comment (randomwangran) [inline CHANGELOG.develop]: config -> configuration?

File: CHANGELOG.develop [hunk 4]
```diff
@@ -715,6 +755,16 @@ is enabled
     - add ~SPC m s~ to save message as draft
     - add ~SPC m f~ to attach file
 - Replace mu4e multiple accounts with contexts (thanks to Allen Li)
+- Refactor to use =spacemacs/add-to-hooks= (thanks to Sylvain Benner)
+- Helm-mu4e integration and key bindings (thanks to Sébastien Bariteau and
```
Comment (randomwangran) [inline CHANGELOG.develop]: Helm-mu4e integration and key bindings - > Add Helm-mu4e integration and key bindings ?

File: CHANGELOG.develop [hunk 5]
```diff
@@ -715,6 +755,16 @@ is enabled
...
+  Sylvain Benner):
+  - ~SPC a M~ to start mu4e
+  - ~SPC m S~ or ~SPC m /~ to search emails (requires helm)
+  - ~SPC m C~ to search contacts (requires helm)
+- Better defaults, async and utility functions (thanks to Peter Hoeg)
+  - Adds binding =C-x m= to compose new message
```
Comment (randomwangran) [inline CHANGELOG.develop]: kbd should use  
tide line?
> ~C-x m~

not 
> =C-x m=

not sure Spacemacs's prefers.

File: CHANGELOG.develop [hunk 6]
```diff
@@ -715,6 +755,16 @@ is enabled
...
+- Maildirs extension is now optional (thanks to JorisE)
```
Comment (randomwangran) [inline CHANGELOG.develop]: Make Maildirs extension optional?

File: CHANGELOG.develop [hunk 7]
```diff
@@ -761,6 +814,33 @@ is enabled
...
+    - ~SPC m b c~ calls =org-babel-check-src-block=
+    - ~SPC m b j~ calls =org-babel-insert-header-arg=
+    - ~SPC m b l~ calls =org-babel-load-in-session=
+    - ~SPC m b i~ calls =org-babel-lob-ingest=
+    - ~SPC m b I~ calls =org-babel-view-src-block-info=
+    - ~SPC m b z~ calls =org-babel-switch-to-session=
+    - ~SPC m b Z~ calls =org-babel-switch-to-session-with-code=
+    - ~SPC m b a~ calls =org-babel-sha1-hash=
+    - ~SPC m b x~ calls =org-babel-do-key-sequence-in-edit-buffer=
+    - ~SPC m b .~ transient state
```
Comment (randomwangran) [inline CHANGELOG.develop]: ~SPC m b .~ enters transient state ?

File: CHANGELOG.develop [hunk 8]
```diff
@@ -777,6 +857,9 @@ is enabled
 - Enable =org-src-tab-acts-natively= (thanks to Sylvain Benner)
 - Fix =org-default-notes-file= precedence (thanks to Troy Hinckley and bmag)
 - Add =ox-hugo= (thanks to Kaushal Modi)
+- Use org-mime provided by melpa (thanks to Amos Bird)
```
Comment (randomwangran) [inline CHANGELOG.develop]: =org-mime= ?

File: CHANGELOG.develop [hunk 9]
```diff
@@ -777,6 +857,9 @@ is enabled
...
+- Restore =evil-org-additional-bindings= in insert mode (thanks to Maximilian
```
Comment (randomwangran) [inline CHANGELOG.develop]: =insert-mode=?

File: CHANGELOG.develop [hunk 10]
```diff
@@ -836,6 +919,7 @@ is enabled
 - Improve =magic-mode-alist= regular expression to detect =react= files
   (thanks to Jan Zdráhal)
 - Use JSX header without breaking React (thanks to Jam Risser)
+- Make auto-completion triggers more friendly (thanks to Cheng,Rong)
```
Comment (randomwangran) [inline CHANGELOG.develop]: Cheng Rong?

Not see other people seperating their family name.

File: CHANGELOG.develop [hunk 11]
```diff
@@ -959,25 +1048,25 @@ is enabled
...
+  Maximilian Wolff, Mikhail Yakutovich, Muneeb Shaikh, Nasser Alshammari, Nasser
+  Alshammari, Niko Felger, Nikolai Myllymäki, nikolaiam, Paul Milla, Paulo
+  Schneider, Pawan Dubey, Paweł Siudak, Phil Pirozhkov, Rafi Khan, Rand01ph,
+  rakyi, Reverend Homer, Robby O'Connor, Sam Pablo Kuper, Saulius Menkevičius,
+  sduthil, Shane Kilkelly, Sid Kapur, Somelauw, Soobin Rho, SteveJobzniak,
+  Swaroop C H, TinySong, Titov Andrey, Thomas de Beauchêne, Tomasz Cichocinski,
+  tzhao11, Vladimir Kochnev, Wieland Hoffmann, Yi Liu, zer09)
 
 *** Release notes summarized
   Thank to: Songpeng Zu
```
Comment (randomwangran) [inline CHANGELOG.develop]: forget your name?

**Comment by sdwolfz** [comment]
@igoralmeida Waiting for you to address @randomwangran's review, after which I'll cherry-pick.

File: CHANGELOG.develop (hunk 9 above)
Comment (yuhan0) [reply CHANGELOG.develop]: insert state *

File: CHANGELOG.develop (hunk 10 above)
Comment (igoralmeida) [reply CHANGELOG.develop]: That's what is written in the commit message. Could be either way, so I'd rather leave it as is.

**Comment by igoralmeida** [comment]
My connectivity will be spotty, I'll probably not be able to work on this for the next few days. Sorry for the delay.

**Comment by sdwolfz** [comment]
Thank you :heart:!
Cherry-picked into `release_notes_0.300` branch, you can safely delete your branch.
Commit: 9a9e52a55b62bfbf97d6951c0dbdc313a7109e9e

**Pull request closed without merging**